src/Symfony/Component/Scheduler/Command/RebootSchedulerCommand.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Scheduler\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * @author Guillaume Loulier <contact@guillaumeloulier.fr>
+ */
+final class RebootSchedulerCommand extends Command
+{
+    private $container;
+
+    /**
+     * @var SymfonyStyle
+     */
+    private $io;
+
+    protected static $defaultName = 'scheduler:reboot';
+
+    public function __construct(ContainerInterface $container)
+    {
+        $this->container = $container;
+
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Reboot a specific Scheduler')
+            ->setDefinition([
+                new InputArgument('name', InputArgument::REQUIRED, 'The name of the scheduler to reboot'),
+            ])
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->io = new SymfonyStyle($input, $output);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $name = $input->getArgument('name');
+
+        $scheduler = $this->container->get('scheduler.hub.'.$name);
```

Comment:
maybe have a scheduler registry instead of injecting the container?

Reply:
I completely agree with this idea, I've implemented this idea, huge thanks 🙂